lib/commands/context.js

```diff
@@ -362,6 +362,26 @@ extensions.onPageChange = async function onPageChange (pageChangeNotification) {
   let newIds = [];
   let newPages = [];
   let keyId = null;
+
+  // Checks if a URL is blacklisted in the 'safariIgnoreWebHostnames' capability
+  function isUrlBlacklisted (url, safariIgnoreWebHostnames) {
+    const blacklistedHosts = (safariIgnoreWebHostnames || '')
+      .split(',')
+      .map((b) => b.trim())
+      .filter((b) => !_.isEmpty(b));
+    for (const blacklistedHost of blacklistedHosts) {
+      if (blacklistedHost === url && url === 'about:blank') {
```

Comment:
this condition looks suspicious to me, should there be `or` instead of `and`?

Reply:
I'll re-write it to:

```
if (blackedlistedHost === 'about:blank' && url === 'about:blank'
```

Reply:
it makes sense to move `about:blank` into a constant

Reply:
I changed it to just `blacklistedHost === url`. It makes sense if the entire blacklisted host and the url are the same, then filter it (this would cover `about:blank`)